src/redux/books/booksSlice.js

```diff
@@ -1,26 +1,45 @@
 /* eslint-disable no-param-reassign */
```

Comment:
- [ ] KIndly  solve linters errors instead of disabling them, as they can help you prevent future bugs.